styles/Home.module.scss

```diff
@@ -14,99 +14,63 @@
 }
 
 .title {
+    @include font-size(xxxxl);
     margin: 0;
     line-height: 1;
-    @include font-size(xxxxl);
     letter-spacing: -1px;
     font-weight: $weight--extrabold;
 
+    @include media-query(large) {
+        @include font-size(xxxxxl);
+    }
```

Comment:
We're not using `xxxxl` anywhere else on the site, so we can change the large font size for xxxxl to be the current large size of xxxxxl, instead of adding another media query and font-size query here. Good placement of the @include